RxJavaTemplate/app/build.gradle

```diff
@@ -104,8 +104,8 @@ android {
         jvmTarget = JavaVersion.VERSION_1_8.toString()
     }
 
-    androidExtensions {
-        experimental = true
+    buildFeatures {
+        viewBinding true
     }
```

Comment:
> Currently we cannot remove kotlin-android-extensions plugin because there is a bug related to Parcelize when applying kotlin-parcelize plugin. It is expected to fix in next kotlin release version. Read here

I think now we could switch to use `apply plugin: 'kotlin-parcelize'` instead of `apply plugin: 'kotlin-android-extensions'` with kotlin 1.4.21, the import will be changed to `import kotlinx.parcelize.Parcelize`

Reply:
@luongvo let me check again, last time I still faced the bug in `kotlin 1.4.21` 🤔

Reply:
@luongvo perhaps upgrading to AS 4.2 fixed this issue, I don't know if this is the reason why the bug no longer happens. Anw, thank you. I removed all `kotlin-android-extension` usage 🙏